server/services/database/profiles.js

```diff
@@ -3,7 +3,7 @@ const config = require("../../config");
 const pool = new Pool(config);
 
 const getAllProfiles = () => {
-	return pool.query("SELECT id, first_name, last_name, phone_number, email FROM profiles")
+	return pool.query("SELECT * FROM profiles")
```

Comment:
Not a change request, but this is an example of where it would be nice to explain in the commit message not only what was done, but why (https://chris.beams.io/posts/git-commit/#why-not-how), to provide context in the log for reviewers and people reading the commit log in future. Also try to avoid saying things like "change the query" since it's not obvious without reading the code which query you mean. If you say something like "Change the getAllProfiles query" that would make it clearer.